packages/schema/lib/appium-config-schema.js

```diff
@@ -288,6 +288,18 @@ export const AppiumConfigJsonSchema = /** @type {const} */ ({
           title: 'webhook config',
           type: 'string',
         },
+        'ssl-certificate-path': {
```

Comment:
`ssl-cert-path` might be less verbose and equally descriptive

Reply:
changed